content/en/platform/corda/4.8/enterprise/operations/monitoring-logging/monitoring-logging-intro.md

```diff
@@ -43,11 +43,11 @@ By default the node log files are stored to the logs subdirectory of the working
 
 It may be the case that you require to amend the log level of a particular subset of modules (e.g., if you’d like to take a closer look at hibernate activity). So, for more bespoke logging configuration, the logger settings can be completely overridden with a Log4j2 configuration file assigned to the log4j.configurationFile system property.
 
-To understand the information in the logs, which data is important, and what can be safely kept as a low priority, you can read the [logging documentation](../../node/operating/node-administration).
+To understand the information in the logs, which data is important, and what can be safely kept as a low priority, you can read the [logging documentation](../../node/operating/node-administration.md).
 
 ## Monitoring - the basics
 
-You can monitor the health and performance of your node [using tools such as Jolokia and Graphite](../../node/operating/monitoring-logging#monitoring-your-node). Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
+You can monitor the health and performance of your node [using tools such as Jolokia and Graphite](../../node/operating/monitoring-logging.html#monitoring-your-node). Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
```

Comment:
```suggestion
You can monitor the health and performance of your node [using tools such as Jolokia and Graphite](../../node/operating/monitoring-logging/overview.html#node-monitoring). Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
```
This is a fix I have in an open PR. I'll merge that after I finish this review, so everything should work out.